media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -95,7 +96,7 @@ PATCHES=(
 
 # limitsdfile related code taken from =sys-auth/realtime-base-0.1
 # with changes as necessary.
-limitsdfile=40-${PN}.conf
+limitsdfile=00-${PN}.conf
```

Comment:
No, we will stick to `40-` to stay in sync with other package. Also, you cannot go below `00-` so this wouldn't be an option at all.

Reply:
But the entire point is that any other file will have higher priority and will override our settings. The higher the number, the higher the priority (but only within the matching rule type e.g. default vs default, group vs. group or UID vs UID). Would you prefer `10-` at least?